### PR TITLE
fix: fix multi-module issue for skaffold lint dockerfile support

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: pkg/skaffold/lint/dockerfiles.go [hunk 1]
```diff
@@ -129,6 +128,11 @@ func GetDockerfilesLintResults(ctx context.Context, opts Options, dockerCfg dock
 		for _, a := range c.Build.Artifacts {
 			if a.DockerArtifact != nil {
 				ws := filepath.Join(workdir, a.Workspace)
+				// TODO(aaron-prindle) HACK - multi-module configs use abs path for a.Workspace vs single module which has rel path
+				// see if there is a built-in/better way of handling this.  This is currently working for multi-module
+				if filepath.IsAbs(a.Workspace) {
+					ws = filepath.Join(a.Workspace)
```
Comment (tejal29) [inline pkg/skaffold/lint/dockerfiles.go:134]: should this be just `ws = a.Workspace`?
```suggestion
					ws = a.Workspace
```

File: pkg/skaffold/lint/dockerfiles.go [hunk 2]
```diff
@@ -129,6 +128,11 @@ func GetDockerfilesLintResults(ctx context.Context, opts Options, dockerCfg dock
...
+				}
```
Comment (tejal29) [inline pkg/skaffold/lint/dockerfiles.go:135]:
```suggestion
				ws := a.Workspace
				// TODO(aaron-prindle) HACK - multi-module configs use abs path for a.Workspace vs single module which has rel path
				// see if there is a built-in/better way of handling this.  This is currently working for multi-module
				if !filepath.IsAbs(ws) {
					ws = filepath.Join(workdir, a.Workspace)
				} 
```

Comment (aaron-prindle) [reply pkg/skaffold/lint/dockerfiles.go]: done

File: pkg/skaffold/lint/dockerfiles.go (hunk 1 above)
Comment (aaron-prindle) [reply pkg/skaffold/lint/dockerfiles.go]: done

**Review by tejal29 — APPROVED** [review]

**Pull request merged**